polyfills/document.querySelector/polyfill.js

```diff
@@ -4,7 +4,7 @@
 
 	function getElementsByQuery(node, selector, one) {
 		var
-		generator = document.createElement(),
+		generator = document.createElement('null'),
```

Comment:
This is valid, it will create an `HTMLUnknownElement` instead of a `HTMLElement`. 
Pasted from [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLUnknownElement):
>The HTMLUnknownElement interface represents an invalid HTML element and derives from the HTMLElement interface, but without implementing any additional properties or methods.